utility/src/main/java/org/oppia/android/util/parser/html/BulletTagHandler.kt

```diff
@@ -23,10 +23,17 @@ class BulletTagHandler : CustomHtmlContentHandler.CustomTagHandler {
   override fun handleClosingTag(output: Editable) {
     output.append("\n")
     output.getSpans(0, output.length, Bullet::class.java).lastOrNull()?.let {
+      val spanEnd = output.getSpanEnd(it)
       val start = output.getSpanStart(it)
-      output.removeSpan(it)
+      // no need for newline if empty span. This fix 'PARAGRAPH span must start at paragraph boundary'.
```

Comment:
I'm not sure I fully understand this comment--could you please rephrase it?

Reply:
If there were empty tags or if there is no end tag. It was crashing with error `PARAGRAPH span must start at paragraph boundary`. But, now as I have formatted the faqs.xml file there are no any empty tags and paragraph tags which would fail here. This is no longer needed so removing it.